lego/apps/users/tests/test_models.py

```diff
@@ -381,3 +381,43 @@ def test_penalty_offset_is_calculated_correctly(self, mock_now):
         self.assertEqual(
             (active.exact_expiration.month, active.exact_expiration.day), (1, 11)
         )
+
+
+class PhotoConsentTestCase(BaseTestCase):
+    fixtures = [
+        "test_users.yaml",
+        "test_abakus_groups.yaml",
+        "test_companies.yaml",
+        "test_events.yaml",
+    ]
+
+    def setUp(self):
+        self.current_semester = "spring"
+        self.current_year = 2021
+        self.test_user = User.objects.get(pk=1)
+
+    def test_update_consent(self):
+        PhotoConsent.objects.create(
+            user=self.test_user,
+            year=self.current_year,
+            semester=self.current_semester,
+            domain="SOCIAL_MEDIA",
+            is_consenting=None,
+        )
+
+        pc = self.test_user.photo_consents.get(
+            year=self.current_year,
+            semester=self.current_semester,
+            domain="SOCIAL_MEDIA",
+        )
+
+        self.assertEqual(pc.user, self.test_user)
+        self.assertEqual(pc.year, self.current_year)
+        self.assertEqual(pc.semester, self.current_semester)
+        self.assertEqual(pc.domain, "SOCIAL_MEDIA")
+        self.assertIsNone(pc.is_consenting)
+
+        pc.is_consenting = True
+        pc.save()
+
+        self.assertEquals(pc.is_consenting, True)
```

Comment:
I would strongly prefer some more tests here. f.ex. Test get consent permissions. Test that the a "new" consent is created when getting consents, that the default values are correct. Test setting different values etc.

Reply:
Done